### PR TITLE
upate descriptions of commands

### PR DESCRIPTION
**Pull request by hongchaodeng**
follow up on https://github.com/oam-dev/kubevela/issues/590

```
Usage:
  vela [flags]
  vela [command]

Available Commands:

  Getting Started:

    init   	Create scaffold for an application
    install	Install Vela Core with built-in capabilities
    up     	Apply an appfile

  Managing Applications:

    config                                                                                    	Manage configurations
    delete APP_NAME                                                                           	Delete an application
    env                                                                                       	Manage environments
    exec [flags] APP_NAME -- COMMAND [args...]                                                	Execute command in a container
    logs                                                                                      	Tail logs for application
    ls                                                                                        	List services of all applications
    port-forward APP_NAME [options] [LOCAL_PORT:]REMOTE_PORT [...[LOCAL_PORT_N:]REMOTE_PORT_N]	Forward local ports to services in an application
    show APP_NAME                                                                             	Show details of an application
    status APP_NAME                                                                           	Show status of an application, including workloads and traits of each service.

  Managing Capabilities:

    cap                              	Manage capability centers and installing/uninstalling capabilities
    template                         	Manage templates
    traits [--apply-to WORKLOAD_NAME]	List traits
    workloads                        	List workloads

  System:

    completion < bash | zsh >	Output shell completion code for the specified shell (bash or zsh...
    dashboard                	Setup API Server and launch Dashboard
    system                   	System management utilities
    version                  	Prints out build version information

Flags:
  -h, --help   help for vela

```

### CODE REVIEW

**Comment by hongchaodeng** [comment]
cc @resouer @wonderflow

File: docs/en/platform-engineers/workload-type.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Extending Traits in KubeVela
+
+In the following tutorial, you will learn how to add a new workload and expose it to users via Appfile.
+
+## Add A New Workload
+
+### Step 1: Install KubeWatch
```
Comment (resouer) [inline docs/en/platform-engineers/workload-type.md:7]: This seems to be a WIP doc?

Comment (hongchaodeng) [reply docs/en/platform-engineers/workload-type.md]: right. removed

**Review by resouer — APPROVED** [review]
LGTM

**Pull request merged**